articles/user-guide/host-programs.md

```diff
@@ -116,7 +116,7 @@ More specifically, the differences revolve around:
 - Specifying the target machine on which to run it
 - How any results are returned
 
-First, we discuss how this is done with the Q# standalone application from the command prompt, and then proceed to using Python and C# host programs.
+In the following sections we will discuss how this is done with the Q# standalone application from the command prompt, and then proceed to using Python and C# host programs.
 We reserve the standalone application of Q# Jupyter Notebooks for last, because unlike the first three, it's primary functionality does not center around a local Q# file.
 
 > [!NOTE]
```

Comment:
We shouldn't use "we" in our docs. Would you be able to rewrite these two sentences (and the note in line 123) to not use that)? Also, if you could check the rest of the article for that, that'd be great. Thank you.

Reply:
Ok, I will. But I think the use of "we" is widespread in the docs :(